packages/psss/src/containers/Panels/AlertsPanel.js

```diff
@@ -88,24 +88,37 @@ AlertsPanelProvider.propTypes = {
 
 export const AlertsPanel = React.memo(() => {
   const { data: panelData, isOpen, setIsOpen } = useContext(AlertsPanelContext);
-  const { organisationUnit: countryCode, period, syndrome, syndromeName } = panelData;
+  const { id: alertId, organisationUnit: countryCode, period, syndrome, syndromeName } = panelData;
   const alert = { period, organisationUnit: countryCode, syndrome };
-
-  const [isModalOpen, setIsModalOpen] = useState(false);
+  const [isArchiveAlertModalOpen, setIsArchiveAlertModalOpen] = useState(false);
+  const [isCreateOutbreakModalOpen, setIsCreateOutBreakModalOpen] = useState(false);
   const [activeIndex, setActiveIndex] = useState(0);
   const notesState = useFetch(getAlertsMessages);
   const activityState = useFetch(getActivityFeed);
   const countryName = useSelector(state => getCountryName(state, countryCode));
 
   const handleChange = option => {
     // TODO handle changes other than creating an outbreak
-    setIsModalOpen(true);
+    switch (option.value) {
+      case 'Archive':
+        setIsArchiveAlertModalOpen(true);
+        break;
+      default:
+    }
   };
 
   const handleClose = useCallback(() => {
     setIsOpen(false);
   }, [setIsOpen]);
 
+  const handleCreateOutbreakModalCLose = useCallback(() => {
```

Comment:
I'd probably rename this (and the other handler below) as follows, but up to you

```suggestion
  const handleCloseCreateOutbreakModal = useCallback(() => {
```